modules/3d-tiles/src/parsers/parse-3d-tile-point-cloud.js

```diff
@@ -200,6 +206,54 @@ export function parseRGB565(rgb565) {
   return [r5, g6, b5];
 }
 
+export function decodeDraco(tile) {
+  const draco = tile.draco;
+  if (draco && draco.buffer) {
+    const data = new DracoParser().decode(draco.buffer);
+    const decodedPositions = data.attributes.POSITION && data.attributes.POSITION.value;
+    const decodedColors = data.attributes.COLOR_0 && data.attributes.COLOR_0.value;
+    const decodedNormals = data.attributes.NORMAL && data.attributes.NORMAL.value;
+    const decodedBatchIds = data.attributes.BATCH_ID && data.attributes.BATCH_ID.value;
+    const isQuantizedDraco = decodedPositions && data.attributes.POSITION.value.quantization;
+    const isOctEncodedDraco = decodedNormals && data.attributes.NORMAL.value.quantization;
+    if (isQuantizedDraco) {
+      // Draco quantization range == quantized volume scale - size in meters of the quantized volume
+      // Internal quantized range is the range of values of the quantized data, e.g. 255 for 8-bit, 1023 for 10-bit, etc
+      const quantization = data.POSITION.data.quantization;
+      const range = quantization.range;
+      tile.quantizedVolumeScale = new Vector3(range, range, range);
+      tile.quantizedVolumeOffset = new Vector3(quantization.minValues);
+      tile.quantizedRange = (1 << quantization.quantizationBits) - 1.0;
+      tile.isQuantizedDraco = true;
+    }
+    if (isOctEncodedDraco) {
+      tile.octEncodedRange = (1 << data.NORMAL.data.quantization.quantizationBits) - 1.0;
+      tile.isOctEncodedDraco = true;
+    }
+    // const styleableProperties = tile.styleableProperties;
+    // for (const name in batchTableProperties) {
+    //   if (batchTableProperties.hasOwnProperty(name)) {
+    //     const property = result[name];
+    //     if (!defined(styleableProperties)) {
+    //       styleableProperties = {};
+    //     }
+    //     styleableProperties[name] = {
+    //       typedArray : property.array,
+    //       componentCount : property.data.componentsPerAttribute
+    //     };
+    //   }
+    // }
+    //
+    // tile.styleableProperties = styleableProperties;
+    tile.positions = decodedPositions || tile.positions;
+    tile.colors = decodedColors || tile.colors;
+    tile.normals = decodedNormals || tile.normals;
+    tile.batchIds = decodedBatchIds || tile.batchIds;
+  }
+  return true;
+}
+/* eslint-enable complexity */
```

Comment:
Just use `// eslint-disable-next-line complexity` in front of function, instead of disabling region of code.